src/chrome/content/rules/CDNetworks.xml

```diff
@@ -1,51 +1,25 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://ssl2.cdngc.net/ => https://ssl2.cdngc.net/: (6, 'Could not resolve host: ssl2.cdngc.net')
-Non-2xx HTTP code: http://cdnetworks.co.jp/ (200) => https://www.cdnetworks.co.jp/ (403)
-Fetch error: http://www.pantherexpress.net/ => https://www.pantherexpress.net/: (60, 'SSL certificate problem: certificate has expired')
-	CDN buckets:
+	Related:
+		cdngc.net.xml
 
-		- clicktalecdn.sslcs.cdngc.net
+	404:
+		www.cdnetworks.co.jp
 
+	Expired：
+		pantherexpress.net
+		www.pantherexpress.net
 
-	Nonfunctional:
-
-		- (www.)cdnetworks.com		(cert: *.pantherssl.com; shows blank page)
-		- landing.cdnetworks.com	(interrupted)
-
-
-	ToDo: discover CDN scheme
+	Failed:
+		landing.cdnetworks.com
 
+	Mismatch:
+		cdnetworks.co.jp
 -->
-<ruleset name="CDNetworks (partial)" default_off='failed ruleset test'>
 
-	<target host="ssl.cdngc.net" />
-	<target host="ssl2.cdngc.net" />
+<ruleset name="CDNetworks">
```

Comment:
www.cn.cdnetworks.com
www.jp.cdnetworks.com
www.kr.cdnetworks.com
emea.cdnetworks.com
fi.cdnetworks.com
de.cdnetworks.com
it.cdnetworks.com
fr.cdnetworks.com
nl.cdnetworks.com
org-www.jp.cdnetworks.com
apache.mirror.cdnetworks.com
centos.mirror.cdnetworks.com

ocsp.kr.cdnetworks.com

cdnetworks.com.sg
www.cdnetworks.com.sg